pkg/yang/entry.go

```diff
@@ -89,6 +89,9 @@ type Entry struct {
 	RPC *RPCEntry // set if we are an RPC
 
 	Augments []*Entry // Augments associated with this entry
+
+	// Extra maps all the unsupported fields to their values
+	Extra map[string][]interface{}
```

Comment:
It can be any of over two dozen types.  This basically lets you extend the compiler in client code.  The compiler gets an interface{} and type asserts it to the type that must be produced by that particular field.